docker/mariadb/00_database-create-geokrety-db.sql

```diff
@@ -1,216 +1,155 @@
-  `zdjecia` smallint(5) UNSIGNED NOT NULL DEFAULT '0',
-  `ost_pozycja_id` int(10) UNSIGNED NOT NULL DEFAULT '0' COMMENT 'position last ruchy id',
-  `ost_log_id` int(10) UNSIGNED NOT NULL DEFAULT '0' COMMENT 'log last ruchy id',
+  `droga` int(10) unsigned NOT NULL,
+  `skrzynki` smallint(5) unsigned NOT NULL,
+  `zdjecia` smallint(5) unsigned NOT NULL DEFAULT '0',
+  `ost_pozycja_id` int(10) unsigned NOT NULL,
+  `ost_log_id` int(10) unsigned NOT NULL,
   `hands_of` int(10) DEFAULT NULL COMMENT 'In the hands of user',
-  `missing` tinyint(1) UNSIGNED NOT NULL DEFAULT '0',
+  `missing` tinyint(1) unsigned NOT NULL DEFAULT '0',
   `typ` enum('0','1','2','3','4') COLLATE utf8_polish_ci NOT NULL,
-  `avatarid` int(10) UNSIGNED NOT NULL DEFAULT '0',
-  `timestamp_oc` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
-  `timestamp` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP
-) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_polish_ci COMMENT='Geokrety';
-
--- --------------------------------------------------------
+  `avatarid` int(10) unsigned NOT NULL,
+  `timestamp_oc` datetime NOT NULL,
+  `timestamp` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+  PRIMARY KEY (`id`),
+  UNIQUE KEY `id` (`nr`) USING BTREE,
+  KEY `owner` (`owner`),
+  KEY `nr` (`nr`),
+  KEY `ost_pozycja_id` (`ost_pozycja_id`),
+  KEY `avatarid` (`avatarid`),
+  KEY `ost_log_id` (`ost_log_id`),
+  KEY `hands_of_index` (`hands_of`),
+  KEY `id_typ` (`typ`) USING BTREE
+) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_polish_ci;
 
---
--- Table structure for table `gk-geokrety-rating`
---
 
 CREATE TABLE `gk-geokrety-rating` (
   `id` bigint(20) NOT NULL COMMENT 'id kreta',
   `userid` bigint(20) NOT NULL COMMENT 'id usera',
-  `rate` float NOT NULL DEFAULT '0' COMMENT 'single rating (number of stars)'
+  `rate` float NOT NULL DEFAULT '0' COMMENT 'single rating (number of stars)',
+  PRIMARY KEY (`id`,`userid`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_polish_ci COMMENT='GK ratings';
 
--- --------------------------------------------------------
-
---
--- Table structure for table `gk-grupy`
---
 
 CREATE TABLE `gk-grupy` (
   `groupid` bigint(20) NOT NULL,
   `kretid` bigint(20) NOT NULL,
   `joined` date DEFAULT NULL
 ) ENGINE=MyISAM DEFAULT CHARSET=utf8 COLLATE=utf8_polish_ci COMMENT='wchich kret belongs to which group';
 
--- --------------------------------------------------------
-
---
--- Table structure for table `gk-grupy-desc`
---
 
 CREATE TABLE `gk-grupy-desc` (
-  `groupid` bigint(20) NOT NULL,
+  `groupid` bigint(20) NOT NULL AUTO_INCREMENT,
   `creator` bigint(20) NOT NULL,
   `created` datetime DEFAULT NULL,
   `private` binary(1) NOT NULL,
   `desc` blob NOT NULL,
-  `name` varchar(128) COLLATE utf8_polish_ci NOT NULL
+  `name` varchar(128) COLLATE utf8_polish_ci NOT NULL,
+  UNIQUE KEY `groupid` (`groupid`),
+  UNIQUE KEY `groupid_2` (`groupid`)
```

Comment:
> >Sounds like extra useless key :)
> 
> No context, I don't understand sorry.

Here

Reply:
Second unique key dropped.


BTW, I never seen the code using this table, probably some code never released.